sdk/go/auto/workspace.go

```diff
@@ -151,6 +152,8 @@ type Workspace interface {
 	ImportStack(context.Context, string, apitype.UntypedDeployment) error
 	// StackOutputs gets the current set of Stack outputs from the last Stack.Up().
 	StackOutputs(context.Context, string) (OutputMap, error)
+	// Install installs the workspace's dependencies.
+	Install(context.Context, *InstallOptions) error
```

Comment:
We're kinda inconsistent about this, but many commands use a variadic argument, such as `...optlist.Option` for passing optional arguments.  I wonder if it would make sense to do the same here, so the user doesn't need to pass in anything if they don't want to oset any options?

Reply:
Ah TIL!